clusterloader2/testing/density/rc.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: {{.Basename}}
+  name: {{.Basename}}
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    name: {{.Basename}}
+  template:
+    metadata:
+      labels:
+        name: {{.Basename}}
+        phase: phase1
+    spec:
+      containers:
+      - image: gcr.io/kubernetes-e2e-test-images/serve-hostname-amd64:1.0
+        imagePullPolicy: IfNotPresent
+        name: {{.Basename}}
+        ports:
+        - containerPort: 80
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 10m
+            memory: "26214400"
+      restartPolicy: Always
```

Comment:
You don't need this. It's the default - https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy